corehq/pillows/case_search.py

```diff
@@ -79,16 +82,26 @@ def transform_case_for_elasticsearch(doc_dict):
     return doc
 
 
+def _format_property(key, value, case_id):
+    if not isinstance(value, str):
+        value = str(value)
+        _assert_string_property(False, f'Case {case_id} has property {key} saved in unexpected format')
```

Comment:
I suspect you're going to get a lot of these. It's situations like this where I wish we had a way to surface this to admins in the domain.

Reply:
Yeah, I am curious to see. It will also be interesting to know how many of these are based on oddly submitted data vs internal bugs. Since it is only case search, it won't catch everything, but should give a decent sense.